data/maps/VermilionCity/text.inc

```diff
@@ -23,29 +23,32 @@ VermilionCity_Text_Machop::
     .string "When we got the takedown notice,\n"
     .string "BOSS told us to demolish the TOWER.\p"
     .string "I'm not sure he understands what\n"
-    .string "'takedown' means in this context.$"
+    .string "“takedown” means in this context.$"
 
 VermilionCity_Text_SSAnneVisitsOnceAYear::
     .string "The ship is gone, so the captain\n"
     .string "isn't around to teach CUT.\p"
-    .string "I wonder how else you could use\n"
+    .string "I wonder how else you could get\n"
     .string "it?$"
 
 VermilionCity_Text_CitySign::
-    .string "VERMILION CITY\n"
-    .string "The Port of Exquisite Sunsets$"
+	.string "VERMILLION CITY\p"
+	.string "We're not totally sure what the\n"
+	.string "color VERMILLION is.\p"
+	.string "It's orange, right?\n"
+	.string "…Yeah, definitely orange.\p"
+	.string "RECOMMENDED POKéDEX COMPLETION: 48\n$"
 
 VermilionCity_Text_SnorlaxBlockingRoute12::
-    .string "NOTICE!\p"
-    .string "ROUTE 12 may be blocked off by a\n"
-    .string "sleeping POKéMON.\p"
-    .string "Detour through ROCK TUNNEL to\n"
-    .string "LAVENDER TOWN.\p"
-    .string "VERMILION POLICE$"
+	.string "NOTICE!\p"
+	.string "VERMILLION RADIO TOWER has been\n"
```

Comment:
switching back and forth between Vermillion and Vermilion is breaking my brain here